<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpielStart</class>
 <widget class="QDialog" name="SpielStart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>184</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stadt Land Fluss</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Copperplate Gothic Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Dein Name: </string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="NameEingabe">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Copperplate Gothic Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonHost">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>100</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Copperplate Gothic Bold</family>
    </font>
   </property>
   <property name="text">
    <string>Spiel hosten</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonBeitreten">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Copperplate Gothic Bold</family>
    </font>
   </property>
   <property name="text">
    <string>Spiel beitreten</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonBeenden">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Copperplate Gothic Light</family>
    </font>
   </property>
   <property name="text">
    <string>Beenden</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
